packages/perseus-editor/src/components/blur-input.tsx

```diff
@@ -45,9 +47,14 @@ class BlurInput extends React.Component<Props, State> {
         this.props.onChange(e.target.value);
     };
 
+    focus() {
```

Comment:
We call `.focus()` on refs for this component in some places. I suspect that was broken code previously and was likely broken by a refactoring from a bare `<input/>` element to this component.